app/assets/stylesheets/seattlerb.css

```diff
@@ -7399,3 +7422,21 @@ abbr.amp {
 /* line 268, /Users/mike/Projects/oss/seattlerb-style.org/vendor/assets/stylesheets/_old_seattlerb_styles.scss */
 #newsletter input[type=text] {
   margin-top: 5px; }
+
+.profile_list {
+  list-style: none;
+  margin-top: 5px;
+}
+
+.book_list {
+  list-style: none;
+  margin-top: 5px;
+}
+
+.book_list li {
```

Comment:
Hi @phiggins, is this how we specify we want the li elements within this book_list class to have these attributes?